importer/tests/test_views.py

```diff
@@ -52,9 +52,8 @@ def test_upload_duplicate(self):
         self.assertTrue(Course.objects.count() == 0)
         self.upload_test_file()
         self.assertTrue(Course.objects.count() == 1)
-        body = self.upload_test_file()
+        self.upload_test_file()
         self.assertTrue(Course.objects.count() == 1)
```

Comment:
Because it no longer is there, even with it running in eager mode, it isn't triggering error messages because it still runs "asynchronously" by returning immediately and the exception is raised outside the form save.  It prioritizes some need for status updates to the user, but that should get cleaned up with the stories coming up in a bit I hope, and it should simplify writing that so we don't have to worry about how to return errors to the user.